Changes after code review

VKuz89 opened the pull request:
changes according to the code review from Victor:
+Program divided into three layers: core, database, ui
+Separate methods in case switch 
+Now Database responsible for product id count

Commit:
Code review comments

VKuz89 committed:
according to latest codereview from Viktor program layout changed:

+Program divided into three layers: core, database, ui
+Separate methods in case switch
+Now Database responsible for product id count

VKuz89 committed:
according to latest codereview from Viktor program layout changed:

+Program divided into three layers: core, database, ui
+Separate methods in case switch
+Now Database responsible for product id count

javagurualex commented:
У тебя есть конфликты с матером. Разреши их пожалуйста

javagurualex commented on src/main/java/com/javaguru/shoppinglist/ui/ShoppingListApplication.java:
```diff
@@ -0,0 +1,116 @@
+        return product;
+    }
+
+    public static String nameEnter (){
```
Это чуть-чуть не по java convention. Используй автоформатирование на классах перед commit/push

javagurualex replied:
Пробел до () не нужен. А нужен после () и перед {

javagurualex commented on src/main/java/com/javaguru/shoppinglist/database/ProductRepository.java:
```diff
@@ -0,0 +1,10 @@
+    Product insert (Product product);
+    Product read (Long productID);
+    Product print ();
+    Product delete (Long productID);
```
Каким образом ты вернёшь Product, если ты его только что удалил?

javagurualex commented on src/main/java/com/javaguru/shoppinglist/database/ProductRepositoryImpl.java:
```diff
@@ -1,21 +1,24 @@
+        product.setId(productIdSequence);
+        localDatabase.put(productIdSequence,product);
+        productIdSequence++;
         return null;
```
Смысл что-то возвращать если это всё равно всега null?

javagurualex replied:
Смысл что-то возвращать если это всё равно всега null?

javagurualex commented on src/main/java/com/javaguru/shoppinglist/database/ProductRepositoryImpl.java:
```diff
@@ -32,4 +35,8 @@ public Product delete(Long productID) {
         return null;
```
Смысл что-то возвращать если это всё равно всега null?

VKuz89 committed:
according to latest codereview from Viktor program layout changed:

+Program divided into three layers: core, database, ui
+Separate methods in case switch
+Now Database responsible for product id count

VKuz89 committed:
according to latest codereview from Viktor program layout changed:

+Program divided into three layers: core, database, ui
+Separate methods in case switch
+Now Database responsible for product id count

VKuz89 committed:
according to latest codereview from Viktor program layout changed:

+Program divided into three layers: core, database, ui
+Separate methods in case switch
+Now Database responsible for product id count

VKuz89 committed:
Merge branch 'master' into changesAfterCodeReview
